index.html

```diff
@@ -0,0 +1,203 @@
+
+                <div class="bg-white/10 p-4 flex md:flex-col justify-center items-center text-center">
+                    <i class="hover:text-[#f28f10] fa-solid fa-podcast fa-2x md:fa-3x"></i>
+                    <h4 class="text-[#f28f10] hover:text-white text-xl px-2 py-2">TALKS</h4>
+                    <p>
+                        Listen to travel hacks from speakers and learn about their experiences.
+                    </p>
+                </div>
+
+                <div class="bg-white/10 p-4 flex md:flex-col justify-center items-center text-center">
+                    <i class="hover:text-[#f28f10] fa-solid fa-user-gear fa-2x md:fa-3x"></i>
+                    <h4 class="text-[#f28f10] hover:text-white text-xl px-2 py-2">WORKSHOP</h4>
+                    <p>
+                        Learn about hiking and traveling to extreme locations safely.
+                    </p>
+                </div>
+
+                <div class="bg-white/10 p-4 flex md:flex-col justify-center items-center text-center">
+                    <i class="hover:text-[#f28f10] fa-solid fa-user-group fa-2x md:fa-3x"></i>
+                    <h4 class="text-[#f28f10] hover:text-white  text-xl px-2 py-2">FORUM</h4>
+                    <p>
+                        Meet new people and join our foum for better travel insights.
+                    </p>
+                </div>
+
+                <div class="bg-white/10 p-4 flex md:flex-col justify-center items-center text-center">
+                    <i class="hover:text-[#f28f10] fa-solid fa-handshake fa-2x md:fa-3x"></i>
+                    <h4 class="text-[#f28f10] hover:text-white  text-xl px-2 py-2 ">CONNECT</h4>
+                    <p>
+                        Connect with people across the globe and learn from them.
+                    </p>
+                </div>
+
+                <div class="bg-white/10 p-4 flex md:flex-col justify-center items-center text-center">
+                    <i class="hover:text-[#f28f10] fa-solid fa-earth-americas fa-2x md:fa-3x"></i>
+                    <h4 class="text-[#f28f10] hover:text-white text-xl px-2 py-2">COURSE</h4>
+                    <p>
+                        Connect with people across the globe and learn from them.
+                    </p>
+                </div>
+
+            </div>
+
+            
+                <a class="hidden md:block" href="#">
+                    <button type="button" class="my-14 hover:text-[#f28f10] underline text-lg hover:underline">
+                        SEE THE WHOLE PROGRAM
+                    </button>
+                </a>
+
+                <a class="block md:hidden" href="#">
+                    <button type="button" class="my-14 text-lg text-white bg-[#f28f10] p-4 active:border-white active:border-2 ">
+                        JOIN PROGRAM
+                    </button>
+                </a>
+            
+
+        </section>
```

Comment:
Kindly make sure that your project sticks as much as possible to the template provided. :eyes: :eyes: 
- Desktop version: The space between cards is different from the template.

Your design
![image](https://user-images.githubusercontent.com/98363075/203324927-e937869f-fa95-447f-a911-cbf264d77967.png)

Template provided
![image](https://user-images.githubusercontent.com/98363075/203325012-a9d1deb6-ddae-431c-a233-88c120e52e79.png)

- Mobile version: The space between cards is different from the template, and the cards should have the same high.

Your design
![image](https://user-images.githubusercontent.com/98363075/203325491-fa82ea04-e2d0-4767-8774-94a79d8fb2d1.png)

Template design
![image](https://user-images.githubusercontent.com/98363075/203325564-e82af344-8d08-4c27-be76-1d73c7284ebf.png)